gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/connection.js

```diff
@@ -69,8 +69,9 @@ class Connection extends EventEmitter {
   constructor(url, options) {
     super();
 
+    options = options || {};
```

Comment:
just peeking in on this PR - I'll let @jorgebay make the final word on this, but I guess the reason we needed to add this line was so that calls further down that use `options` and not `this.options` will work properly? shouldn't we just consistently use `this.options` rather than try to interchangeably use both as well as alter the state of the `options` parameter?

Reply:
I think the safest bet would be to use double assignment, that way is concise and less error prone, dwyt @nivsherf ?

```javascript
this.options = options = options || {};
```

Reply:
Makes sense @jorgebay. I updated the code accordingly.